reports/aksversions/main.py

```diff
@@ -0,0 +1,156 @@
+"""
+Script providing functionality to fetch information
+from Azure for AKS Clusters and calls another script to store in a cosmosdb.
+"""
+# --------
+# Import relevant packages
+# --------
+import os
+import json
+import uuid
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.resource import SubscriptionClient
+from azure.mgmt.containerservice import ContainerServiceClient
+from azure.cosmos import CosmosClient, exceptions
+from cosmos_functions import remove_documents, add_documents
+
+# Define Functions
+def get_minor_version(version_number_str):
+    return float(version_number_str[version_number_str.find('.') + 1: version_number_str.rfind('.')])
+
+# Define Environment variables for Cosmos
+endpoint = os.environ.get("COSMOS_DB_URI", None)
+key = os.environ.get("COSMOS_KEY", None)
+database = os.environ.get("COSMOS_DB_NAME", "reports")
+container_name = os.environ.get("COSMOS_DB_CONTAINER", "aksversions")
+
+###############
+# Script start
+###############
+
+# Authenticate to Azure
+print("Logging into Azure...")
+credential = DefaultAzureCredential()
+
+# Initialize the SubscriptionClient
+subscription_client = SubscriptionClient(credential)
+
+# Get all subscriptions for the tenant
+subscriptions = subscription_client.subscriptions.list()
+
+# Initialize an empty list to hold all the cluster information
+clusters_info = []
+
+# Define your search terms
+search_terms = ['CFT', 'SHAREDSERVICES']
+
+# For each subscription, get all AKS clusters
+print("Using available subscriptions to find AKS clusters...")
+print()
+
+########################################################################
+# Use the list of subscriptions to find all AKS Clusters
+# subscriptions are filtered based on the search_terms so we only search for AKS clusters in subscriptions we own
+########################################################################
+
+for sub in subscriptions:
+    # Get the subscription details
+    sub_details = subscription_client.subscriptions.get(sub.subscription_id)
+
+    # Check if the subscription name contains any of the search terms
+    if any(term in sub_details.display_name for term in search_terms):
+        
+        print("Checking subscription: " + sub_details.display_name)
+        print()
```

Comment:
```suggestion
        print("Checking subscription: " + sub_details.display_name + "\n")
```
little cleaner I think

Reply:
or use `println`